src/secp256k1-compat.ts

```diff
@@ -259,6 +268,9 @@ export function privateKeyTweakMul(
   assertBytes(privateKey, 32);
   assertBytes(tweak, 32);
   let bn = bytesToNumber(tweak);
+  if (bn === 0n) {
```

Comment:
Is `0n` here the "identity` mentioned in the report?

Reply:
Sure.